src/libsync/graphapi/drives.cpp

```diff
@@ -29,8 +29,7 @@ namespace {
 
 const auto mountpointC = QLatin1String("mountpoint");
 const auto personalC = QLatin1String("personal");
-const auto shareC = QLatin1String("virtual");
-
+const auto sharesIdC = QLatin1String("a0ca6a90-a365-4782-871e-d44447bbc668$a0ca6a90-a365-4782-871e-d44447bbc668");
```

Comment:
We should probably document in code where to find this over at the other repository.

Reply:
@micbar @C0rby could you point me to the location its defined?

Reply:
It's defined here: https://github.com/cs3org/reva/blob/edge/pkg/utils/utils.go#L56-L59
And assigned here: https://github.com/cs3org/reva/blob/edge/internal/grpc/services/sharesstorageprovider/sharesstorageprovider.go#L371-L375

Reply:
thx